### ISSUE TITLE
(Not sure how to title, see description)

### ISSUE THREAD

**Issue by LucasZamprogno** [ataraxie/Eyetrack#6]
One thing I found so close to helpful, then annoying: Say I have big window B, and small window S, and S is partially covering B. If I look at B for a while, eventually S fades out, letting me see the content of B it was blocking before. That's great, but when I intuitively go to look at the now visible information, S pops back up and blocks it.

Initially I went to solve this by enabling the setting to bring windows to the front, so that S would end up out of the way completely. The problem then was that if S was completely inside B, once B gets focus and becomes foregrounded, there's no way to get S back using your gaze anymore.

I haven't really thought of a way around this. Only idea I had was to implement some sort of trigger, so that instead of a window coming back into focus after N milliseconds, it comes back into focus when the gaze is on it and the user hits the trigger (e.g. keypress, maybe double blink or something in the future).

**Comment by ataraxie** [comment]
Yeah, I've been noticing this issue too. Both is annoying: S in front of B and S covered by B. We'll have to think about how to solve this is we want this to work in an overlapping environment. I can't come up with a solution right now, but it would need to be some additional trigger as you mentioned.
